internal/cmd/operator-sdk/bundle/validate/cmd.go

```diff
@@ -69,11 +69,11 @@ To list and run optional validators, which are specified by a label selector:
   operatorhub    name=operatorhub           OperatorHub.io metadata validation
                  suite=operatorframework
 
-To validate a bundle with a superset of requirements for Operator Framework:
+To validate a bundle against the requirements for Operator Framework, in addition to the standard bundle validation (Note that it include all optional validators for this suite):
 	
   $ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework
 
-To validate a bundle with a superset of requirements for operatorhub.io specifically:
+To validate a bundle against the requirements for operatorhub.io specifically, in addition to the standard bundle validation:
```

Comment:
```suggestion
To validate a bundle against the operatorhub.io validator, in addition to required bundle validators:
```

Reply:
IMO: 
> To validate a bundle against the requirements for operatorhub.io 

It does not clarify when it should be used. I mean its purpose is exactly what the user is looking for. only `operatorhub.io validator` means that the user would need to know what `operatorhub.io validator` does and its purpose. Besides the name be intuitive I think we need to focus on the end-goal since further validators would use the same. 

> a part of the Operator Framework validator suite,
I think it is not required might bring a misunderstanding

> in addition to required bundle validators: 👍

Reply:
```suggestion
To validate a bundle against the validator for operatorhub.io, in addition to required bundle validators:
```